src/components/SearchResultsPage/SearchResultsPage.test.js

```diff
@@ -181,4 +181,117 @@ describe('<SearchResultsPage />', () => {
         expect(await screen.findByText(errorText))
             .toBeInTheDocument();
     });
+
+    // it("Should display a functional dropdown for sort", async () => {
```

Comment:
Is the intention to merge all of this disabled code?

Reply:
yes